src/sentry/static/sentry/app/views/organizationIntegrations/integrationRepos.jsx

```diff
@@ -46,6 +47,36 @@ export default class IntegrationRepos extends AsyncComponent {
     return this.state.itemList.filter(repo => repo.integrationId === integrationId);
   }
 
+  debouncedSearchRepositoriesRequest = debounce(query => {
+    this.setState(
+      {
+        dropdownBusy: true,
+      },
+      () => this.searchRepositoriesRequest(query)
+    );
```

Comment:
I don't think you really need to do the search as a callback to the `setState`. Since it's not dependent on the state being updated.

Reply:
Actually looking further down even you're already setting `dropdownBusy` to true before making this call